source/frontends/libretro/CMakeLists.txt

```diff
@@ -11,10 +11,55 @@ set(SOURCE_FILES
   rdirectsound.cpp
   retroregistry.cpp
   retroframe.cpp
+  retroframeresources.cpp
   diskcontrol.cpp
   serialisation.cpp
   )
 
+macro(add_resource output source)
+  set(add_resource_command "./resource2inl.sh ../../../resource/${source} ${output}")
+  add_custom_command(
+    OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/${output}.inl"
+    COMMAND bash -c "${add_resource_command}"
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    DEPENDS "../../../resource/${source}"
+  )
+  set(RESOURCE_FILES ${RESOURCE_FILES} "${output}.inl")
+endmacro(add_resource)
```

Comment:
this (and the following lines) runs a script to convert the resource files into arrays which can be referenced at runtime.

Reply:
this could be a good idea to apply to every other frontend. qt already embeds resources. I started once, but eventually lost interest...